oak/server/storage_service.cc

```diff
@@ -0,0 +1,56 @@
+/*
```

Comment:
Maybe let's create a dedicated folder for this, since I expect is going to get quite large but also quite orthogonal to the rest. How about `/oak/server/storage`?

Reply:
Done.